.github/workflows/kitchen.yml

```
@@ -199,7 +199,7 @@ jobs:
           - 'ubuntu-2004'
           - 'ubuntu-2204'
         ruby: ['3.1']
-    runs-on: ubuntu-latest
+    runs-on: ubuntu:20.04
```

with the cgroupns_host option in kitchen.yml file this would need to be 22.04 or latest.
```suggestion
    runs-on: ubuntu-latest
```

---

otherwise I believe you have to remove the kitchen.yml options if reverting to `ubuntu-20.04`.